cfme/infrastructure/virtual_machines.py

```diff
@@ -706,26 +706,29 @@ def power_control_from_provider(self, option):
             raise OptionNotAvailable(option + " is not a supported action")
 
     def migrate_vm(self, email=None, first_name=None, last_name=None,
-                   host_name=None, datastore_name=None):
+                   host=None, datastore=None):
```

Comment:
REQUIRED: please check all dependencies and fix them before renaming something global.
I see at least one unresolved dependency.

Reply:
ok i changed in test_vm_migrate